pkgs/top-level/perl-packages.nix

```diff
@@ -25039,6 +25053,20 @@ with self; {
     };
   };
 
+  TestExit = buildPerlPackage {
+    pname = "Test-Exit";
+    version = "0.11";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/A/AR/ARODLAND/Test-Exit-0.11.tar.gz";
+      hash = "sha256-+9qS034EgdGO68geSNAlIotXGExZstWm9r34cELox7I=";
+    };
+    propagatedBuildInputs = [ ReturnMultiLevel ];
+    meta = {
+      description = "Test whether code exits without terminating testing";
+      license = with lib.licenses; [ artistic1 gpl1Plus ];
+    };
+  };
+
```

Comment:
Also commit this separately as `perlPackages.TestExit: init at 0.11`

Reply:
I have rearanged the commits in git so they appear in a correct order based on dependencies:
```
[akegalj@x220:~/projects/nixpkgs]$ git log --pretty=format:%s HEAD^~2..HEAD
perlPackages.AppSqitch: 1.4.0 -> 1.4.1
perlPackages.TestExit: init at 0.11
perlPackages.ReturnMultiLevel: init at 0.08
```